### ISSUE TITLE
Simulation switching does not clean up state from previous simulation/recording

### ISSUE THREAD

**Issue by Ragzouken** [IRL2/nanover-protocol#303]
Something relevant in https://github.com/IRL2/nanover-protocol/pull/225 but doesn't quite address the issue. Interactions and selections should not persist between switches. Avatars etc wouldn't hurt. Box position?

**Comment by Mohamed-Dhouioui** [comment]
Is this related to #302 ?

**Comment by Ragzouken** [comment]
> Is this related to #302 ?

Yes I think #302 is because of this -- I didn't see that you already created it
